src/components/checkbox/checkbox.ts

```diff
@@ -126,7 +129,9 @@ export class MdCheckbox implements ControlValueAccessor {
 
   hasFocus: boolean = false;
 
-  constructor(private _renderer: Renderer, private _elementRef: ElementRef) {}
+  constructor(private _renderer: Renderer, private _elementRef: ElementRef) {
+    this.change = this._change.asObservable();
```

Comment:
Why are we initializing the observable in the constructor? It should (works in slide-toggle) work without it as well.

```ts
@Output('change') private _change: EventEmitter<boolean> = new EventEmitter<boolean>();
change: Observable<boolean> = this._change.asObservable();
```